tests/broadcast.log

```diff
@@ -0,0 +1 @@
+DEVICE MAC ADDRESS: ([0-9a-fA-F]{2}:){5}[0-9a-fA-F]{2}
```

Comment:
What is the format of broadcast.log; what type of regex ?

Reply:
it is just standard regex, this file can be consumed by greentea/htrun.
So we can add this test in ci to run these examples on targets (partially), rather than just compile the example